core/storage/collection/gae_models.py

```diff
@@ -140,8 +140,8 @@ class CollectionRightsModel(base_models.VersionedModel):
     owner_ids = ndb.StringProperty(indexed=True, repeated=True)
     # The user_ids of users who are allowed to edit this collection.
     editor_ids = ndb.StringProperty(indexed=True, repeated=True)
-    # The user_ids of users who are allowed to translate this collection.
-    translator_ids = ndb.StringProperty(indexed=True, repeated=True)
+    # The user_ids of users who are allowed to voiceover this collection.
+    voice_artist_ids = ndb.StringProperty(indexed=True, repeated=True)
```

Comment:
We should keep the `translator_ids` over here with a comment similar to what we have done in other models.

(We don't need migration here as this value isn't used in the collection.)

Reply:
Done.